torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -448,6 +448,7 @@ RegisterOperators reg(
      Operator(
          FunctionSchema(
              "aten::warn",
+             "",
```

Comment:
Can you just replace this with:
"aten::warn(str message, *, int stacklevel=2)"

It must have been added before we supported the kwarg_only annotation.

Reply:
hm that fails with https://ci.pytorch.org/jenkins/job/pytorch-builds/job/py2-devtoolset7-rocmrpm-centos7.5-test/4357/console . Reverting and using just the name for now.